src/Sulu/Bundle/LocationBundle/Geolocator/GeolocatorLocation.php

```diff
@@ -14,165 +14,153 @@
 use Sulu\Component\Util\TextUtils;
 
 /**
- * Data object representing a location returned
- * by a geolocator.
+ * Data object representing a location returned by a geolocator.
  */
 class GeolocatorLocation
 {
     /**
      * ID of this location (according to the geolocation vendor).
      *
-     * @var mixed
+     * @var string
      */
     protected $id;
 
     /**
-     * Title to display.
-     *
-     * @var string
+     * @var string|null
      */
     protected $displayTitle;
 
     /**
-     * @var string
+     * @var string|null
      */
     protected $street;
 
     /**
-     * @var string
+     * @var string|null
      */
     protected $number;
 
     /**
-     * @var string
+     * @var string|null
      */
     protected $code;
 
     /**
-     * @var string
+     * @var string|null
      */
     protected $town;
 
     /**
-     * @var string
+     * @var string|null
      */
     protected $country;
 
     /**
-     * @var float
+     * @var float|null
```

Comment:
shouldnt locations at leas have a `longitude` and `latitude`?  🤔

Reply:
no for example polygon results have no long/lat